tests/arc/topt_no_cursor.nim

```diff
@@ -101,6 +105,7 @@ try:
           `=destroy`(splitted)
 finally:
   `=destroy_1`(lan_ip)
+-- end of expandArc ------------------------
```

Comment:
Why would this line be new? Something is messed up here.

Reply:
but here is expected, it should be present at each injectDestructorCalls

Reply:
Is this a testament bug then? How come it ever was green before?

Reply:
it's messed up before, https://github.com/nim-lang/Nim/commit/f32ffb6ed821cc01e52c48181a4caa15e73c0362 line 109 followed `-- end of expandArc ------------------------` before, and then it doesn't.

Reply:
That's a severe testament issue. Can you investigate?

Reply:
sure, I'll try

Reply:
it's just mis used test, https://github.com/nim-lang/Nim/pull/20791